app/models/position.rb

```diff
@@ -12,6 +12,16 @@ class Position < ApplicationRecord
 
   validates_presence_of :course_code, :openings, :session
   validates :openings, numericality: { only_integer: true }
+
+  def as_json(_options = {})
+    super(
+      except: [:session_id],
+      include: {
+        session: { only: %i[id year round semester] },
+        instructors: { only: %i[id first_name last_name] }
```

Comment:
Should we include `utorid`?

Reply:
i can't think of a use case for it, this can always be modified later though